django/core/validators.py

```diff
@@ -283,8 +283,11 @@ def ip_address_validators(protocol, unpack_ipv4):
                          % (protocol, list(ip_address_validator_map)))
 
 
-def int_list_validator(sep=',', message=None, code='invalid'):
-    regexp = _lazy_re_compile('^\d+(?:%s\d+)*\Z' % re.escape(sep))
+def int_list_validator(sep=',', message=None, code='invalid', allow_negative=False):
+    if allow_negative is True:
+        regexp = _lazy_re_compile('^(-)?\d+(?:%s(-)?\d+)*\Z' % re.escape(sep))
```

Comment:
I'd rather build regex string with interpolation so as not to repeat the other logic.